runtime/dart_controller.h

```diff
@@ -19,6 +19,7 @@ class DartController {
   DartController();
   ~DartController();
 
+  void RunFromKernel(const uint8_t* buffer, size_t size);
```

Comment:
Can we make this return `tonic::DartErrorHandleType`. I think that is the mechanism we are going to use to correctly propagate `exit` codes.